proseco/tests/test_catalog.py

```diff
@@ -46,7 +47,24 @@ def test_get_aca_catalog_49531():
     assert len(aca.fids) == 0
 
 
-@pytest.mark.skipif('not HAS_SC_ARCHIVE', reason='Test requires starcheck archive')
+@pytest.mark.skipif(not HAS_SC_ARCHIVE, reason='Test requires starcheck archive')
+def test_get_aca_catalog_20603_with_supplement():
```

Comment:
This test looks to assume mag updates for a specific obsid.  I was just thinking that one possible path to promotion might be to promote all the agasc code and then promote the supplement separately (which I think would have this test failing).  If we are planning on promoting the supplement before the code or together I suppose this is fine.

Reply:
Fixed